### PR TITLE
Recipe

### PR DESCRIPTION
**Pull request by kosher9**
Changes made on this pull request : 
- [ ] Added interface skeleton for recipe
- [ ] Setup controller for recipe

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -3,16 +3,21 @@ class RecipesController < ApplicationController
...
+    # Should allow the user to delete it's own recipes
+    # Show details of the recipe
     @recipe = Recipe.find(param[:id])
   end
 
   # GET /recipes/new
   def new
+    # The logged-in user should be able to create a new recipe
     @recipe = Recipe.new
   end
```
Comment (sentayhu19) [inline app/controllers/recipes_controller.rb:23]: - [ ] I am getting this error when I try to create a new recipe.
I think the recipes controller create function should create based on users.
```
user = current_user
    @recipe = Recipe.new(recipe_params)
    @recipe.user = user
```
Screenshot 2022-07-28 100408

**Review by sentayhu19 — CHANGES_REQUESTED** [review]
Hi @kosher9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (kosher9) [reply app/controllers/recipes_controller.rb:23]: No, you can't rught now. I nade that pull request just to be able to use the same routes as you and the design.

**Comment by sentayhu19** [comment]
okay if that so lets merge it to dev @kosher9

**Review by sentayhu19 — APPROVED** [review]
## Hi @kosher9 👋,   ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**